docs/photoniq/fingerprinting/fingerprinting-metrics.md

```diff
@@ -0,0 +1,18 @@
+---
+sidebar_position: 40
+title: Fingerprinting Metrics
+---
+
+The tracking metrics below quantify the scale of Digital Fingerprinting and can help identify usage trends.
+
+## Accuracy Rate
+
+Percentage of visitors correctly identified.
```

Comment:
This we need to work on.  Let me see if I can get @lohaniabhishek thoughts on this.

Reply:
It would also be great if we could tell users how to access the metrics.

Reply:
We pretty much use the same thing as described [here](https://dev.fingerprint.com/docs/understanding-our-995-accuracy). Please see if it can be leveraged

Reply:
I used our friend to summarize the website and give me a one-paragraph synopsis.  I then pick out the stuff below.  Do you think this describes accuracy?

_Fingerprinting achieves accuracy by combining various browser attributes to create a unique user identifier. This amalgamation results in a high recognition rate for returning web visitors._

And this is mine that I came up with:

_The accuracy rate is the confidence of the unique Visitor ID being that returning user using the various unique attributes of a user._

If these sound good, please cleanup to make it sound better.

Reply:
I took a crack at it, please review.

How does a user access these metrics?

Reply:
That works for me.

These metrics are accessed via the metrics API.